extensions/apple-reminders/src/menu-bar.tsx

```diff
@@ -178,7 +178,7 @@ export default function Command() {
                 <MenuBarExtra.Item
                   title="Open Reminder"
                   onAction={() => open(reminder.openUrl, "com.apple.reminders")}
-                  icon={{ fileIcon: "/System/Applications/Reminders.app" }}
+                  icon={Icon.AppWindow}
```

Comment:
Could also be `AppWindowList` or `BulletPoints`

Reply:
Did you consider this?

Reply:
I actually disagree on that one. With this new icon, I'm not sure if opening it would open it in Raycast or in the Reminders app. I'd stick with the Reminders native icon to stay consistent with the other commands.